fmx_annealer.py

```diff
@@ -109,17 +105,64 @@ def govStateSet(stateStr, configStr = 'Robot'):
     return
 
 from ophyd import PVPositioner, PVPositionerPC, Device, Component as Cpt, EpicsMotor, EpicsSignal, EpicsSignalRO
```

Comment:
@JunAishima  do you have any idea what happened here? My code doesn't work, I think because Cpt doesn't correspond to Component (possibly among other issues).

Reply:
Or not, I'm very confused...

Reply:
hmm, not sure. those names seem to be part of the main Ophyd namespace. what is the exception you get, or however you know it's not working?

Reply:
Ah, nevermind, dumb mistake on my part wrt. the imports. My exception is the program complaining that it can't subscribe to inStatus, maybe this is because I haven't wrapped it in the Run Engine?

Reply:
sounds likely - I haven't dug through the code to see what happens in that kind of case. you can probably do a bluesky.plan_stubs.wait instead of using the status's wait directly